prometheus-to-sd/translator/stackdriver.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package translator
+
+import (
+	"fmt"
+
+	"github.com/golang/glog"
+	v3 "google.golang.org/api/monitoring/v3"
+
+	"k8s.io/contrib/prometheus-to-sd/config"
+)
+
+func SendToStackdriver(service *v3.Service, config *config.GceConfig, ts []*v3.TimeSeries) {
+	if len(ts) == 0 {
+		glog.Warningf("No metrics to send to Stackdriver")
```

Comment:
Because this either means that you have a bad configuration or the monitored component doesn't behave properly.